Code/Mantid/Build/Jenkins/buildscript

```diff
@@ -146,19 +146,21 @@ if [[ "$BUILDPKG" == true ]]; then
   # Set some variables relating to the linux packages
   if [[ $(uname) != 'Darwin' ]]; then
     # Use different suffix for linux builds
-    if [[ ${JOB_NAME} == *release* ]]; then
-      PACKAGINGVARS="${PACKAGINGVARS} -DENVVARS_ON_INSTALL=True -DCPACK_SET_DESTDIR=ON"
-    elif [[ ${JOB_NAME} == *master* ]]; then
-      PACKAGINGVARS="${PACKAGINGVARS} -DENVVARS_ON_INSTALL=False -DCPACK_SET_DESTDIR=OFF"
+    if [[ ${JOB_NAME} == *master* ]]; then
       PACKAGE_SUFFIX="nightly"
     elif [[ ${JOB_NAME} == *pvnext* ]]; then
-      PACKAGINGVARS="${PACKAGINGVARS} -DENVVARS_ON_INSTALL=False -DCPACK_SET_DESTDIR=OFF"
       PACKAGE_SUFFIX="mantidunstable-pvnext"
     else
-      PACKAGINGVARS="${PACKAGINGVARS} -DENVVARS_ON_INSTALL=False -DCPACK_SET_DESTDIR=OFF"
       PACKAGE_SUFFIX="unstable"
     fi
 
+    # Only unsuffixed release builds create envvars scripts
+    if [[ ${JOB_NAME} == *release* && -z "$PACKAGE_SUFFIX" ]]; then
```

Comment:
`-z "$PACKAGE_SUFFIX"` will never be True because in the if statement above you always define `PACKAGE_SUFFIX`.